pele/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright Zapata Computing, Inc. All rights reserved.
+
+# Dockerfile for the base QE docker image
+FROM zapatacomputing/z-qml:nightly
+WORKDIR /app
+USER root
+
+#pip install -r pele/requirements.txt fails on install hungarian
+# install needed dependencies manually instead
+RUN apt-get install gcc gfortran g++ && \
+    pip install cython sqlalchemy networkx && \
+    git clone https://github.com/pele-python/pele.git && \
```

Comment:
Could this line actually be replaced with `git clone -b dev https://github.com/hsim13372/pele.git && \`? 

I added some functions to analyze landscapes and also fixed some issues with imports in the dev branch of my fork.